xwiki-platform-core/xwiki-platform-web/src/main/webapp/resources/js/xwiki/table/livetable.js

```diff
@@ -1239,3 +1239,54 @@ function init() {
 // End XWiki augmentation.
 return XWiki;
 }(XWiki || {}));
+
+
+// Used for "link": "editor" property.
+// TODO: move the entire definition of the function in jquery when #livetablecallback
+// from macros.vm, where it is called, will also be moved.
+var editProperty = function(fullname, classname, fieldname, cb) {
+  require(['jquery'], function($) {
+    let doc = new XWiki.Document('LiveTableResults', 'XWiki');
+    let durl = doc.getURL('view', 'xpage=plain&page=' + fullname + '&classname=' + classname + '&fieldname='
+      + fieldname);
+    let editorModal = (`#editorModal`);
+    $('.xwiki-livetable').children().append(editorModal);
+    $('#propEditor').modal('show');
+    $.ajax({
+      url : durl,
+      complete : function(e) {
+        $('#propEditor').on('shown.bs.modal', function () {
+          $('#propEditorForm').html(e.responseText);
+        })
+        $('#propEditorSave').one('click', function() {
+          let value = getValue($('#propEditorForm'), fieldname);
+          let surl = doc.getURL('view', 'xpage=plain&page=' + fullname + '&classname=' + classname + '&fieldname='
+            + fieldname + '&value=' + value);
+          $.ajax({
+            url : surl,
+            complete : function(e) {
+              if(e.responseText.search('SAVED') != -1) {
```

Comment:
I don't know much this part of the code but I find it a bit weird to rely on such a string to assess that an AJAX request was performed correctly. Now maybe there's no other easy solution here to check it?

Reply:
Indeed, there could be used "success" and "error" in the ajax request, instead of  "complete".